UITests/AuthenticationManagerTests.swift

```diff
@@ -148,14 +122,11 @@ class AuthenticationManagerTests: KIFTestCase {
     }
 
     func testChangeRequirePasscodeInterval() {
-        setPasscode("1337", interval: .Immediately)
+        PasscodeUtils.setPasscode("1337", interval: .Immediately)
 
         openAuthenticationManager()
         tester().tapViewWithAccessibilityLabel("Require Passcode, Immediately")
 
-        tester().waitForViewWithAccessibilityLabel("Enter Passcode")
-        enterPasscodeWithDigits("1337")
-
```

Comment:
Looks like you kept the passcode entry when changing the interval -- does that mean these two lines should stay?

I tried running the test myself to find out, but I'm just getting a `Constant is not finite!  That's illegal.  constant:nan` error at [this line](https://github.com/mozilla/firefox-ios/pull/1689/files#diff-7498dcd4a47815cd1c1ae8ffa08618ffL19). Are you seeing issues running the tests too?